packages/strapi-admin/admin/src/containers/Admin/NavTopRightWrapper.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const style = {
+  position: 'fixed',
+  top: '0',
+  right: '0',
+  display: 'flex',
+  zIndex: '1050',
```

Comment:
Hmm, do we really need to do that?

Reply:
Some legacy I didn't spend much time on the DOM... IF I recall it's because the nav also has a hug z-index..

Reply:
F*ck... keep it like that if there is no other way